inlong-dashboard/src/locales/en.json

```diff
@@ -538,6 +538,7 @@
   "pages.GroupDetail.Info.DataStreams": "Data streams",
   "pages.GroupDetail.Info.Submit": "Submit",
   "pages.GroupDetail.Info.SubmittedSuccessfully": "Submitted successfully",
+  "pages.GroupDetail.Info.SubmittedWarn": "Please create data source and sink first",
```

Comment:
What if the user wants to customize push and consume data without source and sink?

Reply:
InLong is not an MQ service, a data stream must contain a source and sink, `auto push` and `auto push` are kinds of source and sink.

Reply:
Okay~